packages/wallet-contract/schema/v1/wallet-contract-documents.json

```diff
@@ -0,0 +1,55 @@
+{
+  "txMetadata": {
```

Comment:
maybe add `    "documentsMutable": false,
    "canBeDeleted": true,`

Reply:
Yes, I agree with this.